Dockerfile

```diff
@@ -185,8 +185,14 @@ RUN set -ex && \
     rm -rf /var/lib/apt
 COPY --from=builder /src/build/release/bin /usr/local/bin/
 
+# Create monero user
+RUN adduser --system --group --disabled-password monero && \
+	mkdir -p /wallet /home/monero/.bitmonero && \
+	chown -R monero:monero /home/monero/.bitmonero && \
+	chown -R monero:monero /wallet
```

Comment:
/wallet ? In the root directory ?

Reply:
It is in /wallet directory in current Dockerfile. I did not change this. But yes, we can probably do it a better way but as this is a container I do not think this is a big deal. Anyway, feel free to correct this if you think this is better. My point is to launch monerod process as a regular user and not as root in the current Dockerfile.

Reply:
If it's already that way, then it's fine.